playbooks/tinc.yaml

```
@@ -0,0 +1,10 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Install and configure VPN.
+---
+- name: Install and configure VPN
+  hosts: all
+  roles:
+    - role: tina_pm.third_party.tinc
+      when: inventory_hostname in groups['tinc_servers']
+      tags: tinc
```

This was a hack to run the role in the same play that targetted all hosts, but here you can just change the `hosts` value
```suggestion
  hosts: tinc_servers
  roles:
    - role: tina_pm.third_party.tinc
      tags: tinc
```